pkg/cmd/server/admin/create_bootstrap_project_template.go

```diff
@@ -0,0 +1,87 @@
+package admin
+
+import (
+	"bytes"
+	"errors"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/spf13/cobra"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/openshift/origin/pkg/api/latest"
+	"github.com/openshift/origin/pkg/project/registry/projectrequest/delegated"
+)
+
+const (
+	DefaultProjectTemplateFile                = "openshift.local.config/master/project.json"
+	CreateBootstrapProjectTemplateCommand     = "create-bootstrap-project-template"
+	CreateBootstrapProjectTemplateFullCommand = "openshift admin " + CreateBootstrapProjectTemplateCommand
```

Comment:
For example, this could either "openshift admin" or "osadm" and which to use depends on how this was called.